## Fourth Year Committee Report
Sarah Stevens, McMahon Lab  
Meeting: October 26th, 2017 in MSB 5503  

I am interested in understanding how bacterial populations diversify and form species.
My work aims to better understand the forces that shape bacterial population structure using the lakes as a model system.
The McMahon Lab metagenomic time series of many lakes is an ideal dataset for investigating how wild bacterial populations change through time.

### Research Progress
The manuscript I submitted last year has been accepted to ISMEJ.

<!-- Add in summary of paper here -->


#### New Work
I continue to be interested in discerning the forces shaping wild bacterial populations and would like to turn my focus from the whole genomes of populations to their gene content.

<!-- Add binning diagram -->

*Are there related sequence-discrete populations in bog lakes (Crystal Bog, Mary Lake, Trout Bog)?  How closely related are they?  Do they share a common gene pool?  Are there genes present in one lake but not the other for these closely related populations? Is there an ecological or physical barrier to recombination between these two lakes?*
I have been working on getting bins for Crystal Bog and Mary Lake over the past year.
My current challenges are dereplicating them, since we likely have recovered the same genomes from multiple timepoints, and classifying them, as past methods did not scale up.
I will also bin additional genomes from Trout Bog using new assemblies of the individual time points.
Once I have binned, dereplicated, classified genomes from the three bog lakes I can start comparing them.
With these genomes, I will search for very closely related genomes and quantify how similar they are across their genomes both the percentage and identity of shared genomic content.
<!-- Continue work on this section, fleshing it out and increasing clarity -->
For genomes that are very closely related (tentatively about >95% nucleotide identity across a shared genome content of at least 50% of their genome), I will also investigate if the associated populations share a common gene pool between the two lakes.
By mapping the metagenomes from one lake to MAGs from the other, I will identify if there are regions or genes that are present in only one of the lakes and if the diversity of shared genes is different between the two lakes.
I will search for patterns among the shared or absent genes that may be explained by the different environments of the two lakes.
I will also look for evidence of a barrier to recombination between these allopatric populations.


### Publications
Garcia, S. L.\*, **Stevens, S. L. R.**\*, Crary, B., Martinez-Garcia, M., Stepanauskas, R., Woyke, T., Tringe, S. G., Andersson, S., Bertilsson, S., Malmstrom, R.,  McMahon, K. D. (_accepted to ISMEJ_). Contrasting patterns of genome-level diversity across distinct co-occurring populations.
Bendall, M. L.\*, **Stevens, S. L. R.**\*, Chan, L.-K., Malfatti, S., Schwientek, P., Tremblay, J., Schackwitz, W., Martin, J., Pati, A., Bushnell, B., Froula, J., Kang, D., Tringe, S. G., Bertilsson, S., Moran, M. A., Shade, A., Newton, R. J., McMahon, K. D., Malmstrom, R. R. (2016). Genome-wide selective sweeps and gene-specific sweeps in natural bacterial populations. ISMEJ. doi:[10.1038/ismej.2015.241](http://dx.doi.org/10.1038/ismej.2015.241) **featured article**  
^\*Equal\ contributors^



### Poster Presentations
<!-- Add in GRC poster here -->


### Professional Development
<!-- Update this section! -->
* Software/Data Carpentry
    + Taught 4 Software Carpentry Workshops (2 on-campus, 2 off-campus)
    + Helped with 2 Data Carpentry Workshops on-campus
    + Helped with Instructor Training on-campus
* Computational Biology, Ecology, and Evolution(ComBEE) support group - Started Nov. 2014
* ComBEE Python Study Group - Started Dec. 2014
* ComBEE R Study Group - Started Fall 2015
* MDTP Steering Committee - 2015-2016

### Awards
* UW-Madison Dept. of  Bacteriology Travel Award - Spring 2017
